netbox/extras/dashboard/widgets.py

```diff
@@ -37,10 +35,7 @@
 def get_content_type_labels():
     return [
         (content_type_identifier(ct), content_type_name(ct))
-        for ct in ContentType.objects.filter(
-            FeatureQuery('export_templates').get_query() | Q(app_label='extras', model='objectchange') |
-            Q(app_label='extras', model='configcontext')
-        ).order_by('app_label', 'model')
+        for ct in ContentType.objects.public().order_by('app_label', 'model')
```

Comment:
The public query looks like it is opening up to a wider number of models then what these existing queries do?  For example this one is does it have objectchange but the new one is not marked private, especially for plugins I think this could be different?   Not sure what effect that would have.

Reply:
The use of `public()` does return some additional object types as opposed to the original query, but I think that's ultimately what we want. The old query was pieced together by including things we identified as desirable, versus excluding those that we know are not.